build-scripts/meson-postinstall.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import sysconfig
+from compileall import compile_dir
+from os import environ, path
+from subprocess import call
+
+prefix = environ.get('MESON_INSTALL_PREFIX', '/usr/local')
```

Comment:
I don't think you need to provide a fallback here, this should always be set by meson.

Reply:
Both [Authenticator](https://gitlab.gnome.org/GNOME/gnome-tweaks/-/blob/master/meson-postinstall.py) and [GNOME Tweaks](https://gitlab.gnome.org/GNOME/gnome-tweaks/-/blob/master/meson-postinstall.py) post-build scripts are doing it.. I must admit I don't really know why, but it seemed like a good practice.

Reply:
In doubt, I'll keep it. Doesn't seem to be much problematic anyway.